controls/radautocompletebox/getting-started.md

```diff
@@ -10,31 +10,39 @@ position: 1
 
 # Getting Started
 
-Getting started with __RadAutoCompleteBox__ is pretty straightforward.
+This tutorial will walk you through the creation of a sample application that contains __RadAutoCompleteBox__.
+
+* [Assembly References](#assembly-references)
+* [Add RadAutoCompleteBox to the Project](#add-radautocompletebox-to-the-project)
+* [Populate with Data](#populate-with-data)
+* [Customize the Control](#customize-the-control)
+* [Set a Theme](#set-a-theme)
+
+## Assembly References
 
 >In order to use __RadAutoCompleteBox__ in your project you have to add a reference to the following assemblies:
 >	* __Telerik.Windows.Controls__
 >	* __Telerik.Windows.Controls.Input__
 
-## Adding RadAutoCompleteBox to the project
+## Add RadAutoCompleteBox to the Project
 
-It is possible to include the control in your projects by either dragging it from the toolbox in Visual Studio and Expression Blend or by manually creating the control.
+You can include the control in your projects by either dragging it from the toolbox in Visual Studio and Blend or by manually creating the control.
 
-The following example shows how to create a __RadAutoCompleteBox__ control in the code behind and in XAML:
+The following examples show how to create a __RadAutoCompleteBox__ control in the code-behind and in XAML:
 
-#### __[C#] Creating RadAutoCompleteBox__
+#### __[C#] Example 1: Creating a RadAutoCompleteBox__
 
 {{region cs-radautocompletebox-getting-started-0}}
 	RadAutoCompleteBox autoCompleteBox = new RadAutoCompleteBox();
 {{endregion}}
 
-#### __[VB] Creating RadAutoCompleteBox__
+#### __[VB] Example 1: Creating a RadAutoCompleteBox__
 
 {{region vb-radautocompletebox-getting-started-0}}
 	Dim autoCompleteBox As New RadAutoCompleteBox()
 {{endregion}}
 
-#### __[XAML] Creating RadAutoCompleteBox__
+#### __[XAML] Example 1: Creating a RadAutoCompleteBox__

```

Comment:
I know the snippet shows the same as the previous two, but the Example counting convention was that "Xaml examples are different than code examples, and c# code examples uses the same number as VB examples. For example:

[C#] Example 1
[VB.NET] Example 1
[C#] Example 2

We can discuss this further because this was established long ago, back when there were several teams and I don't know what information was provided across the teams. This is what I remember.